craft_providers/lxd/lxd_instance.py

```diff
@@ -419,10 +419,32 @@ def mount(
         *,
         host_source: pathlib.Path,
         target: pathlib.PurePath,
-        device_name: Optional[str] = None,
     ) -> None:
         """Mount host source directory to target mount point.
 
+        Checks first to see if already mounted.
+        The source will be mounted as a disk named "disk-{target.as_posix()}".
+
+        :param host_source: Host path to mount.
+        :param target: Instance path to mount to.
+
+        :raises LXDError: On unexpected error.
+        """
+        self.mount_with_device_name(
```

Comment:
is there a reason to not impement mount_with_device_name here?

Reply:
Because the concept of a `device_name` doesn't exist for Multipass.

Reply:
I think you did not understand me, I am not asking to change the signature of mount, but instead, in this LXD specific implementation instead of calling self.mount_with_device_name, just implement that here

Reply:
I think I understand and am now calling the underlying lxc calls instead.

Reply:
Third try!  Now I have removed the `mount_with_device_name()` method.